Expensify/App
# Reset multiline on submit

fixes: https://github.com/Expensify/ReactNativeChat/issues/327

### Problem
When submitting a comment with more than one line it never resets the lines to just one line

### Testing
1. Add a comment with just one line. Verify that the comment is added
1. Add a comment with 2 or more lines. Verify that after submitting the comment box goes back to being just one line.
